src/assets/styles/_component.scss

```diff
@@ -0,0 +1,61 @@
+:host {
+  background-color: var(--calcite-app-background);
```

Comment:
I'm not sure if we should set a default `background-color` since there are a few components that already overwrite it and set it to transparent.

Reply:
There should be [a relevant var](https://github.com/Esri/calcite-app-components/blob/master/src/assets/styles/_color.scss#L28) from _color.

Reply:
yes, so we should remove this and let each component decide the background?